Automatic Validation ended with:
> 2025-03-28T14:53:28.3041144Z ##[error] Result: Failed
 2025-03-28T14:53:28.3095462Z ##[error] Result: Failed

(Automated response - build 1029.)

--- a/manifests/w/Wondershare/PDFPasswordRemover/1.5.3/Wondershare.PDFPasswordRemover.installer.yaml
+++ b/manifests/w/Wondershare/PDFPasswordRemover/1.5.3/Wondershare.PDFPasswordRemover.installer.yaml
@@ -1,9 +1,8 @@
-# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-3
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+# Created with YamlCreate.ps1 v2.4.5 $debug=AUSU.CRLF.7-5-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
 
 PackageIdentifier: Wondershare.PDFPasswordRemover
-PackageVersion: 1.5.3 # null
-MinimumOSVersion: 10.0.0.0
+PackageVersion: 1.5.3
 InstallerType: inno
 Scope: machine
 UpgradeBehavior: install
@@ -13,4 +12,4 @@ Installers:
   InstallerSha256: FB2A6C0E052014038FAF7D369132C0E02B47E0A7D4B417E41BD0B277CD1A87E6
   ProductCode: '{1719FAD6-2F6A-4F5E-BF2B-1F6F6F1E3806_PasswordRemover}_is1'
 ManifestType: installer
-ManifestVersion: 1.1.0
+ManifestVersion: 1.10.0

--- a/manifests/w/Wondershare/PDFPasswordRemover/1.5.3/Wondershare.PDFPasswordRemover.locale.en-US.yaml
+++ b/manifests/w/Wondershare/PDFPasswordRemover/1.5.3/Wondershare.PDFPasswordRemover.locale.en-US.yaml
@@ -1,8 +1,8 @@
-# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-3
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+# Created with YamlCreate.ps1 v2.4.5 $debug=AUSU.CRLF.7-5-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
 
 PackageIdentifier: Wondershare.PDFPasswordRemover
-PackageVersion: 1.5.3 # null
+PackageVersion: 1.5.3
 PackageLocale: en-US
 Publisher: Wondershare Software
 PublisherUrl: https://www.wondershare.com/
@@ -15,9 +15,9 @@ License: Proprietary
 LicenseUrl: https://www.wondershare.com/company/end-user-license-agreement.html
 Copyright: Copyright © 2016 Wondershare. All rights reserved.
 CopyrightUrl: https://www.wondershare.com/company/terms_conditions.html
-ShortDescription: Remove password protection from PDF, making it free to copy, edit, and print 
+ShortDescription: Remove password protection from PDF, making it free to copy, edit, and print
 Description: Wondershare PDF Password Remover is a small and easy-to-use PDF cracker to help you remove PDF password and restrictions in seconds.
-# Moniker: 
+# Moniker:
 Tags:
 - crack
 - cracker
@@ -29,8 +29,10 @@ Tags:
 - remover
 - unlock
 - unlocker
-# Agreements: 
 ReleaseNotes: fix some bugs
-# ReleaseNotesUrl: 
+# ReleaseNotesUrl:
+# PurchaseUrl:
+# InstallationNotes:
+# Documentations:
 ManifestType: defaultLocale
-ManifestVersion: 1.1.0
+ManifestVersion: 1.10.0

--- a/manifests/w/Wondershare/PDFPasswordRemover/1.5.3/Wondershare.PDFPasswordRemover.locale.zh-CN.yaml
+++ b/manifests/w/Wondershare/PDFPasswordRemover/1.5.3/Wondershare.PDFPasswordRemover.locale.zh-CN.yaml
@@ -1,8 +1,8 @@
-# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-3
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+# Created with YamlCreate.ps1 v2.4.5 $debug=AUSU.CRLF.7-5-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.10.0.schema.json
 
 PackageIdentifier: Wondershare.PDFPasswordRemover
-PackageVersion: 1.5.3 # null
+PackageVersion: 1.5.3
 PackageLocale: zh-CN
 Publisher: Wondershare Software
 PublisherUrl: https://www.wondershare.com/
@@ -17,7 +17,7 @@ Copyright: Copyright © 2016 Wondershare. All rights reserved.
 CopyrightUrl: https://www.wondershare.com/company/terms_conditions.html
 ShortDescription: 移除 PDF 密码保护，从而能够自由复制、编辑和打印 PDF 文件。
 Description: Wondershare PDF Password Remover 是一款轻巧易用的 PDF 破解器，帮助您在几秒种内移除 PDF 密码和限制。
-# Moniker: 
+# Moniker:
 Tags:
 - pdf
 - 密码
@@ -25,8 +25,10 @@ Tags:
 - 移除
 - 解密
 - 解锁
-# Agreements: 
-# ReleaseNotes: 
-# ReleaseNotesUrl: 
+# ReleaseNotes:
+# ReleaseNotesUrl:
+# PurchaseUrl:
+# InstallationNotes:
+# Documentations:
 ManifestType: locale
-ManifestVersion: 1.1.0
+ManifestVersion: 1.10.0

--- a/manifests/w/Wondershare/PDFPasswordRemover/1.5.3/Wondershare.PDFPasswordRemover.yaml
+++ b/manifests/w/Wondershare/PDFPasswordRemover/1.5.3/Wondershare.PDFPasswordRemover.yaml
@@ -1,8 +1,8 @@
-# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-3
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+# Created with YamlCreate.ps1 v2.4.5 $debug=AUSU.CRLF.7-5-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.10.0.schema.json
 
 PackageIdentifier: Wondershare.PDFPasswordRemover
-PackageVersion: 1.5.3 # null
+PackageVersion: 1.5.3
 DefaultLocale: en-US
 ManifestType: version
-ManifestVersion: 1.1.0
+ManifestVersion: 1.10.0

